content/Hello yaaf - Juan Manuel Ramallo.md

```diff
@@ -0,0 +1,73 @@
+## Yet another active form
```

Comment:
```suggestion
## Do you use form objects in your Rails apps?
```

Reply:
I find the title "Yet another active form" to be not entirely expressive or catchy at all, what do you think of changing the title to the suggestion above? @andrea-fajardo